fitbenchmarking/resproc/plots.py

```diff
@@ -0,0 +1,117 @@
+"""
+Higher level functions that are used for plotting the best fit plot and a starting guess plot.
+"""
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+from fitbenchmarking.utils import create_dirs
+
+
+class plot(object):
+    """
+    Class providing plotting functionality.
+    """
+
+    def __init__(self, problem, options, count, group_results_dir):
+        self.problem = problem
+        self.options = options
+        self.count = count
+        self.group_results_dir = group_results_dir
+        self.figures_dir = create_dirs.figures(group_results_dir)
+
+    def default_plot_options(self):
+        """
+        Defines defaults plot options for matplotlib
+        """
+        self.markers = "x"
+        self.colour = "k"
+        self.linestyle = '--'
+        self.z_order = 2
```

Comment:
Some of these aren't specific to the dataplot and could be separate, also markersize seems to be the only one that's not set before running it.

Reply:
done